source/server/admin/ungrouped_stats_request.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include "envoy/server/admin.h"
+
+#include "source/server/admin/stats_params.h"
+#include "source/server/admin/stats_render.h"
+#include "source/server/admin/stats_request.h"
+#include "source/server/admin/utils.h"
+
+#include "absl/container/btree_map.h"
+#include "absl/types/variant.h"
+
+namespace Envoy {
+namespace Server {
+
+class UngroupedStatsRequest
```

Comment:
Add comment explaining that this is for treating each stat on its own, as opposed to the Grouped version which groups by tag-name.